tests/codeception.dist.yml

```diff
@@ -27,6 +27,9 @@ modules:
         browser: chrome
         url: '${project.local.protocol}://${project.local.hostname}/'
         window_size: maximize
+        capabilities:
```

Comment:
Can you point me to what this does?

Reply:
This just moves the alert behavior of the WebDriver to config https://www.selenium.dev/documentation/legacy/json_wire_protocol/#capabilities-json-object